pyproject.toml

```diff
@@ -25,14 +25,14 @@ classifiers = [
 urls = { Homepage = "https://github.com/janosh/pymatviz" }
 requires-python = ">=3.9"
 dependencies = [
-    "matplotlib>=3.6.2,!=3.9.1",
-    "numpy>=1.21.0,<2",
```

Comment:
Perhaps it's too early to drop NumPy 1.x support completely? This might reduce our compatibility with other packages? Another package might require `numpy < 2` and pip would have trouble resolving such conflict.

Reply:
good point!